ui/tests/acceptance/allocation-detail-test.js

```diff
@@ -76,37 +76,58 @@ module('Acceptance | allocation detail', function(hooks) {
   });
 
   test('each task row should list high-level information for the task', async function(assert) {
-    const task = server.db.taskStates.where({ allocationId: allocation.id }).sortBy('name')[0];
```

Comment:
This was only checking the first task, but since there can only be one IP per task, I need at least two tasks to check both V4 and V6 IPs, so this now loops over the tasks.